src/app/components/radio-button/radio-button.component.html

```diff
@@ -0,0 +1,44 @@
+<div>
+  <div class="tds-headline-02 tds-u-pb2">Radio Button</div>
+  <form
+    (submit)="onSubmit($event)"
+    #formRef
+  >
+    <ng-container *ngFor="let fieldset of processedData">
+      <fieldset
+        [attr.key]="fieldset.name"
+        class="tds-u-pb2"
+      >
+        <legend class="tds-headline-05 tds-u-pb1">{{ fieldset.title }}</legend>
+        <ng-container *ngFor="let field of fieldset.fields">
+          <tds-radio-button
+            [name]="fieldset.name"
+            [value]="field.name"
+            [radio-id]="field.id"
+            [attr.required]="fieldset.required"
```

Comment:
Is there a value in setting for example [attr.required] here rather than just: [required]?